src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2700,19 +2700,19 @@
     <value>Toggle group sort direction</value>
   </data>
   <data name="NewTabDescription" xml:space="preserve">
-    <value>Open a new tab</value>
+    <value>Open a new tab in the same window</value>
   </data>
   <data name="NavigateBackDescription" xml:space="preserve">
-    <value>Navigate backward</value>
+    <value>Navigate to the previous folder</value>
```

Comment:
There are page types that are not folders (home, release notes, settings). I would prefer reverting to reduce the changes to localization files, but if you feel it's not clear enough, we could use the term 'location'.